lstchain/scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -101,8 +96,7 @@ def main():
         #
         # produce drs4 time calibration file
         #
-
-        time_file = f"{output_dir}/time_calibration.Run{run}.0000.hdf5"
+        time_file = f"{output_dir}/time_calibration.Run{run:05d}.0000.hdf5"
```

Comment:
We use `.hdf5` for calibration files and `.h5` for DL1 and DL2 files. Should we unify that or it is better to keep it like this?